### DISCUSSION TITLE
ScriptReference/Time-time

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#481] [Notes]
# ScriptReference/Time-time

https://docs.unity3d.com/ScriptReference/Time-time.html

**Comment by radiatoryang** [comment]
Recommend using `Time.timeSinceLevelLoad` or `Time.timeAsDouble` instead. Imagine a user leaving the game open for 6+ hours ... maybe they really like the game, maybe demoing the game in a kiosk, etc. The floating point imprecision will start building up, and may introduce bugs.
